### ISSUE TITLE
Dependencies need updates

### ISSUE THREAD

**Issue by brunocascio** [FaridSafi/react-native-gifted-chat#520]
#### Issue Description
`"undefined is not an object (evaluating '_react2.default.PropTypes.shape')"`

`react-native-parsed-text` is still using React.PropTypes, which is not compatible with RN 0.46

#### Steps to Reproduce / Code Snippets

Try with RN 0.46

#### Expected Results

Updates `react-native-parsed-text` dependency in order to use with RN > 0.44 (as documentation said)

#### Additional Information

* React Native version: 0.46.*
* react-native-gifted-chat version: 0.2.5
* Platform(s) (iOS, Android, or both?): both

**Comment by abeltje1** [comment]
Same issue here. Why do the docs specify that it's working when it's clearly not?

**Comment by brunocascio** [comment]
@abeltje1 I have made a PR: https://github.com/FaridSafi/react-native-gifted-chat/pull/523

**Comment by abeltje1** [comment]
Amazing. much love to you!

**Comment by Maniae** [comment]
Why has the PR not been accepted yet ?

**Comment by lrettig** [comment]
Can we please push this through? As it stands this library is incompatible with the current version of RN.

**Comment by mauron85** [comment]
All deps seems to be fixed in their repos.
Diff against #b12d426b4ec568187b6c031a2c2f17e83473437a  (current master)

```
diff --git a/package.json b/package.json
index bbaba97..57040ea 100644
--- a/package.json
+++ b/package.json
@@ -41,9 +41,9 @@
     "moment": "2.18.1",
     "prop-types": "15.5.10",
     "react-native-communications": "2.2.1",
-    "react-native-invertible-scroll-view": "1.0.0",
-    "react-native-lightbox": "oblador/react-native-lightbox#c84a8543d4511fe6a44c3d7820747c9c1bddd875",
-    "react-native-parsed-text": "0.0.18",
+    "react-native-invertible-scroll-view": "1.1.0",
+    "react-native-lightbox": "0.7.0",
+    "react-native-parsed-text": "0.0.19",
     "shallowequal": "1.0.2",
     "uuid": "3.1.0"
   }

```

**Comment by brunocascio** [comment]
@mauron85 Yep! Yesterday I made a commit with these changes.

Thanks!

**Comment by brunocascio** [comment]
Merged and released (v0.2.8)
